Sources/Random/CryptoRandom.swift

```diff
@@ -6,7 +6,11 @@ public final class CryptoRandom: Random {
 
     public func bytes(_ num: Int) -> Bytes {
         var random = Bytes(repeating: 0, count: num)
-        RAND_bytes(&random, Int32(num))
+		guard RAND_bytes(&random, Int32(num)) == 1 else {
+			// If the requested number of random bytes couldn't be read,
+			// we need to fail fast and hard.
+			abort()
```

Comment:
Just saw the messages in #development. I can't believe the `bytes` method doesn't throw... >_<.

Can we move the protocol to throwing and add a non-throwing extension that gets a deprecation warning and has a `try!` ?